src/main/resources/db/changelog/changes/07-create-books-categories-table.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-categories-table
+      author: vadymhrnk
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
It is not necessary to add nullable: false if you have a primaryKey

Reply:
agree